CONTRIBUTING.md

```diff
@@ -196,10 +195,10 @@ To keep naming patterns consistent across the project, naming patterns are enfor
 In order to simplify developing within the project, library recommendations have been set
 and should be followed.
 
-| Scenario | Recommended                                   | Rationale        |
-| ----------------------------------------------------------------------------|
-| Hashing  | ["hashing/fnv"](https://pkg.go.dev/hash/fnv)  | The project adopted this as the default hashing method due to the efficiency and is reasonable for non cryptographic use |
-| Testing  | Use `t.Parallel()` where possible             | Enabling more test to be run in parallel will speed up the feedback process when working on the project.                 |
+| Scenario 	| Recommended                                  	| Recommended                                                                                                              	|
```

Comment:
This header now contains Recommended twice, is this expected?

Reply:
No... That was my oversight of trying to use an online formatter because I never remember what markdown engine wants.